src/Widgets/DataImageWidget/DataImageWidgetEditingConfig.ts

```diff
@@ -0,0 +1,34 @@
+import { provideEditingConfig } from 'scrivito'
+import { DataImageWidget } from './DataImageWidgetClass'
+import Thumbnail from './thumbnail.svg'
+
+provideEditingConfig(DataImageWidget, {
+  title: 'Data Image',
+  thumbnail: Thumbnail,
+  attributes: {
+    alignment: {
+      title: 'Alignment',
+      description: 'Default: Left',
+      values: [
+        { value: 'left', title: 'Left' },
+        { value: 'center', title: 'Center' },
+        { value: 'right', title: 'Right' },
+      ],
+    },
+    alternativeText: {
+      title: 'Alternative text',
+      description: 'Brief description of what the image is about.',
+    },
```

Comment:
* [ ] I think for a data image this makes no sense b/c at editing time the content of the image is unknown, so there is no decision basis. How about always using `alt=""`?